### PR TITLE
Add support for containMatchingElement

### PR DESCRIPTION
**Pull request by superzadeh**
Fixes https://github.com/producthunt/chai-enzyme/issues/85

- add support for containsMatchingElement
- add support to run unit tests on Windows using cross-env
- add support for VSCode editor (in .vscode folder)
- exclude npm-debug.log* files from git
- require enzyme version 2.3.0 or above (when containsMatchingElement was introduced)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/.DS_Store": true,
+    "**/build": true
+  }
+}
```
Comment (ljharb) [inline .vscode/settings.json]: i feel like IDE settings don't belong in repos, and regardless, should be added in a separate PR if they're to be added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+0.6.2 / February 10 2017
```
Comment (ljharb) [inline CHANGELOG.md]: best to say "unreleased" - requiring enzyme 2.3 is technically a breaking change, so this would at a minimum be 0.7.0, but it's not for individual PRs to decide version numbers.

File: package.json [hunk 3]
```diff
@@ -21,13 +21,14 @@
...
-    "test:unit": "NODE_ENV=test mocha test/**/*.test.js",
+    "test:unit": "cross-env NODE_ENV=test mocha test/**/*.test.js",
+    "test:watch": "npm run test:unit -- --watch",
     "test": "npm run test:lint && npm run test:unit"
   },
   "peerDependencies": {
     "chai": "3.x",
     "cheerio": "0.19.x || 0.20.x || 0.22.x",
-    "enzyme": "1.x || 2.x",
+    "enzyme": "1.x || ^2.3.0",
```
Comment (ljharb) [inline package.json]: this line makes it a breaking change.

Comment (superzadeh) [inline package.json]: It does indee. Ddo you need me to do something specific in the PR about this?

Comment (ayrton) [inline package.json]: Update the changelog like @ljharb suggested please

File: .vscode/settings.json (hunk 1 above)
Comment (ayrton) [inline .vscode/settings.json]: Agree, can we remove these from this PR please

File: CHANGELOG.md (hunk 2 above)
Comment (superzadeh) [inline CHANGELOG.md]: Done, see 4b7b1c8

File: .vscode/settings.json (hunk 1 above)
Comment (superzadeh) [inline .vscode/settings.json]: Done, see 13f3164.

Do you want me to open another PR for these settings? They can help other people to get started more quickly when contributing to the project (allow debugging unit tests out of the box), and the maintenance required for these settings is minimal.

File: package.json (hunk 3 above)
Comment (superzadeh) [inline package.json]: Added a notice for the breaking change in the changelog.md, see 93fa730

**Comment by benjamine** [comment]
🎉  thanks, this is great!

**Comment by levithomason** [comment]
Would love to use the features in this PR, any idea on a release date?

**Comment by ayrton** [comment]
@levithomason see issue #150, won't be before mid April that I will be able to cut a release

**Comment by levithomason** [comment]
Got it, thanks for the info.
